sdk/file_sdk/OPCFileFormat/SConscript

```diff
@@ -5,7 +5,6 @@ Import('file_env')
 src_files = Glob('*.cpp')
 
 opc_env = file_env.Clone()
-opc_env.Append(CPPPATH = Split('#sdk/file_sdk'))
```

Comment:
opc doesn't need this include path for Common/blabla.h?

Reply:
it needs it. I moved it to file_sdk/SConscript.
So no need to include this in every file_sdk project

---
In reply to: [139609287](https://github.com/AzureAD/rms-sdk-for-cpp/pull/163#discussion_r139609287) [](ancestors = 139609287)